ij/src/main/kotlin/org/jetbrains/academy/test/system/ij/formatting/FormattingUtil.kt

```diff
@@ -17,3 +22,18 @@ fun PsiFile.checkIfFormattingRulesWereApplied() {
     val formattedCode = ApplicationManager.getApplication().runReadAction<String> { text }
     assert(originalCode.trimIndent() == formattedCode.trimIndent()) { "The code after formatting should be:${System.lineSeparator()}$formattedCode${System.lineSeparator()}Please, apply code formatting refactoring to the code." }
 }
+
+fun PsiFile.checkIfOptimizeImportsWereApplied() {
```

Comment:
I didn't quite understand you, do you mean that instead of one `KotlinUnusedImportInspection` in this function take a list of inspections and check each one?

Reply:
As I understood, Anastasia suggested to create an utility function that takes as parameters a list of inspections names and piece of code to run these inspections on. And it should return true is all checks passed, and false otherwise. It's useful check that can be used in another tests then.